src/commands/install.yml

```diff
@@ -0,0 +1,30 @@
+description: "Install Ruby within a build. To be used in a Linux distro with Apt available."
+parameters:
+  version:
+    description: "Ruby version."
+    type: string
+steps:
+  - run:
+      name: "Prep for RVM"
+      command: |
+        # Determine if sudo is neccessary
+        SUDO=""
+        if [[ $EUID -ne 0 ]]; then
+          SUDO=sudo
+        fi
+
+        $SUDO apt-get update && $SUDO apt-get install -y ruby-full
+  - run:
+      name: "Install Ruby v<< parameters.version >> via RVM"
+      command: |
+        # Fix for retrieving GPG over IPv6 issue
+        mkdir ~/.gnupg && echo "disable-ipv6" >> ~/.gnupg/dirmngr.conf
+        gpg --keyserver hkp://pool.sks-keyservers.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
+
+        curl -sSL "https://get.rvm.io" | bash -s stable
+        adduser root rvm
```

Comment:
happen to know if on a machine without a root user if this might require sudo?

Reply:
Every Linux system has a root user. Earlier I don't assume that it's being used though so I shouldn't assume it here. Good catch. I'll get this fixed.